### PR TITLE
Addisie Marigeta damse endalew

### PR DESCRIPTION
**Pull request by Addisie**

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
very nice, thank you!

File: new/PRS14417MarigetaDamseEndalew.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደምፀ፡ እንዳለው፡</persName>                 
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Damḍa ʾƎndālaw</persName>
+                    <persName><roleName type="title">Marigetā</roleName></persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">He got Dǝggʷā accreditation at <placeName ref="LOC1887BetaLe">Beta Lǝḥem
+                    </placeName> in <date when="1985"/> since then he has been teaching Dǝggʷā at<placeName ref="LOC2386DabraT">
+                            Dabra Tābor ʾIyasus church.</placeName> </occupation>                    
+                    <occupation type="literary">He is a private professional scribe since
```
Comment (eu-genia) [inline new/PRS14417MarigetaDamseEndalew.xml:55]: is ---> has been

File: new/PRS14417MarigetaDamseEndalew.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደምፀ፡ እንዳለው፡</persName>                 
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Damḍa ʾƎndālaw</persName>
+                    <persName><roleName type="title">Marigetā</roleName></persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">He got Dǝggʷā accreditation at <placeName ref="LOC1887BetaLe">Beta Lǝḥem
+                    </placeName> in <date when="1985"/> since then he has been teaching Dǝggʷā at<placeName ref="LOC2386DabraT">
```
Comment (eu-genia) [inline new/PRS14417MarigetaDamseEndalew.xml:53]: space after at

File: new/PRS14417MarigetaDamseEndalew.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS14417MarigetaDamseEndalew.xml:55]: add 
`as of 2024`

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you!

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
thank you, all is good, but unfortunately the application is down so I would like to wait with merging any PRs for the moment

File: new/PRS14417MarigetaDamseEndalew.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <change who="AY" when="2024-04-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደምፀ፡ እንዳለው፡</persName>                 
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Damḍa ʾƎndālaw</persName>
+                    <persName><roleName type="title">Marigetā</roleName></persName>
```
Comment (CarstenHoffmannMarburg) [inline new/PRS14417MarigetaDamseEndalew.xml:50]: In contrast to common practice and to distinguish them from the names, we usually write titles in small letters: marigetā

File: new/PRS14417MarigetaDamseEndalew.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">He got Dǝggʷā accreditation at <placeName ref="LOC1887BetaLe">Beta Lǝḥem
+                    </placeName> in <date when="1985"/> since then he has been teaching Dǝggʷā at <placeName ref="LOC2386DabraT">
+                            Dabra Tābor ʾIyasus church.</placeName> </occupation>                    
+                    <occupation type="literary">He has been a private professional scribe since
+                        <date when="1980"/> as of 2024.</occupation>
```
Comment (CarstenHoffmannMarburg) [inline new/PRS14417MarigetaDamseEndalew.xml:56]: comma before as

File: new/PRS14417MarigetaDamseEndalew.xml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ደምፀ፡ እንዳለው፡</persName>                 
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Damḍa ʾƎndālaw</persName>
+                    <persName><roleName type="title">Marigetā</roleName></persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">He got Dǝggʷā accreditation at <placeName ref="LOC1887BetaLe">Beta Lǝḥem
+                    </placeName> in <date when="1985"/> since then he has been teaching Dǝggʷā at <placeName ref="LOC2386DabraT">
```
Comment (CarstenHoffmannMarburg) [inline new/PRS14417MarigetaDamseEndalew.xml:53]: Split sentence: "... . Since then"

**Pull request merged**